### DISCUSSION TITLE
Please add ability to extend size of card code editor

### DISCUSSION THREAD

**Discussion by cniry** [home-assistant/frontend#16720] [Dashboard] [answered]
It would be nice to be able to extend width of yaml editor in some `Card configuration` -> `Show code editor` (by some button). Even on wide screen monitor, it have only 492px.

**Comment by karwosts** [comment, marked as answer]
Do you mean like this?

![wide-card-editor](https://github.com/home-assistant/frontend/assets/32912880/54a824b3-5fb8-434c-9d68-3b5cfdbe90de)

**Comment by cniry** [comment]
It's perfect, thank you :+1:

**Reply by karwosts** [reply]
Wish it wasn't so secret, its very useful but almost like a hidden easter egg 😛
